packages/node/package.json

```diff
@@ -20,17 +20,27 @@
     "tsc-test": "tsc -p tsconfig.test.json && tsc-alias -p tsconfig.test.json",
     "compile-test": "npm run clean && npm run tsc-test",
     "test-coverage": "npm run compile-test && c8 mocha",
-    "test": "npm run compile-test && NODE_ENV=test mocha 'dist/**/*.test.js'"
+    "test": "npm run compile-test && NODE_ENV=test mocha 'dist/**/*.test.js'",
+    "postinstall": "patch-package"
   },
   "dependencies": {
-    "@libp2p/bootstrap": "^10.0.11",
-    "@libp2p/interface-peer-id": "^2.0.2",
-    "@libp2p/tcp": "9.0.17",
-    "@waku/core": "0.0.27",
-    "@waku/relay": "0.0.10",
-    "@waku/sdk": "0.0.23",
```

Comment:
Going to try and only use sdk package / postinstall

Reply:
leave the tcp one, this is required for the node module